tabs/search/frame.lua

```diff
@@ -422,7 +422,7 @@ do
 			add_form_component()
 		end
     end
-    input:SetOptions({'and', 'or', 'not', unpack(aux.keys(filter_util.filters))})
+    input:SetOptions({'not', unpack(aux.keys(filter_util.filters))})
```

Comment:
I don't understand this change, what does it do exactly? I think the reason I added them here is so that they appear first in the suggestions (since the filter_util filters are in a map which will return them in some random order)

Reply:
> I think the reason I added them here is so that they appear first in the suggestions (since the filter_util filters are in a map which will return them in some random order) so ideally that should be preserved somehow.

Yes, you are right about the display order. I came up with how to save the order of displaying filters and even set and change it at any desired moment. I will make a commit a little later.

> Also, arity is actually already supported, not sure if you're aware of this, it's not very elegant but you can enter for example and3 as an operator.
No, I did not know about it. This is now undocumented and not entirely obvious. In ReadMe to the addon, I saw the use of the arity and to use it at first generated a request without arity and then add an arity in the search bar in the desired locations. You do not like my implementation? It seems to me this is the best solution. If an additional editbox is displayed - it is immediately obvious that you can write something there. It is only necessary to actualize a little ReadMe to the addon by adding a description of this feature.